media/js/firefox/firstrun/firstrun_quantum.js

```diff
@@ -31,10 +36,8 @@
 
         var onVerificationComplete = function () {
             scene.dataset.signIn = 'true';
-            document.getElementById('wave1').addEventListener('animationend', function(event) {
-                if (event.animationName === 'Expand1') {
-                    window.location.href = redirectDest;
-                }
+            document.getElementById('white-overlay').addEventListener('transitionend', function() {
```

Comment:
Changed this because "wave1" no longer animates on all platforms, but the white overlay does transition.

Reply:
in that case "wave1" id can be removed, it wasn't used for anything else.